scripts/us_bea/states_gdp/README.md

```diff
@@ -1,22 +1,90 @@
-# Importing US Bureau of Economic Analysis quarterly GDP data per US state
+# Importing US Bureau of Economic Analysis quarterly GDP data per US state into Data Commons
 
-## Raw Data
-- Available for download at [https://www.bea.gov/data/gdp/gdp-state](https://www.bea.gov/data/gdp/gdp-state).
+Author: fpernice-google
+
+## Table of Contents
+
+1. [About the Dataset](#about-the-dataset)
+1. [Import Procedure](#Import-Procedure)
+
+## About the Dataset
+
+### Download URL
+
+ZIP file is available for download from [https://www.bea.gov/data/gdp/gdp-state](https://www.bea.gov/data/gdp/gdp-state).
```

Comment:
```suggestion
A ZIP file is available for download from https://www.bea.gov/data/gdp/gdp-state.
```

There are many varieties of markdown.  The one I know will display a URL as a URL without the need to add a link.  https://guides.github.com/features/mastering-markdown/ > Links seems to confirm this.